src/index.js

```diff
@@ -0,0 +1,39 @@
+// ----------------- Import Required Elements
```

Comment:
- [ ] OPTIONAL You have many comments in your code. In general, your code is very readable, so there is no needs to make comments on each section. Your code is talking about what happening by itself. So, removing unneeded comments will make your code looks clear 👍